### PR TITLE
Extra lines from TODO Association CIQUAL/AGB

### PR DESCRIPTION
**Pull request by bernardlemoullec**
https://docs.google.com/spreadsheets/d/0B2Z-7cB-sMWOY0hybHNoY1pqaXoyMi1nblI1V1g0aTlvVXFF
Work in Progress (as of May 18th)

### CODE REVIEW

**Comment by teolemon** [comment]
taxonomies/categories.txt needs to be rebuilt; 1589794009 vs 1589366010
taxonomies/ingredients.txt needs to be rebuilt; 1589292919 vs 1589217024
taxonomies/ingredients_processing.txt needs to be rebuilt; 1588958438 vs 1587825760
taxonomies/labels.txt needs to be rebuilt; 1589445322 vs 1588179416
taxonomies/minerals.txt needs to be rebuilt; 1589191921 vs 1585580196
taxonomies/nucleotides.txt needs to be rebuilt; 1589191921 vs 1585580196

File: taxonomies/categories.txt [hunk 1]
```diff
@@ -45831,6 +45862,14 @@ ciqual_food_code:en:25164
 ciqual_food_name:en:Osso buco
 ciqual_food_name:fr:Osso buco
 
+<en:Meals with meat
+<en:meat dishes, no garnish
```
Comment (teolemon) [inline taxonomies/categories.txt:45866]:
```suggestion
```

Comment (bernardlemoullec) [reply taxonomies/categories.txt:45866]: Thanks for the info, will be fixed in the next commit

**Comment by teolemon** [comment]
``` 
Errors in the categories taxonomy definition:
ERROR - fr:soupe already is a synonym of soupes (en:soups) - cannot make a synonym of soupe for en:soups
ERROR - fr:quiches already is a synonym of tartes-salees (en:quiches) - cannot make a synonym of quiches for en:quiches
ERROR - en:puff-pastry already is a synonym of puff-pastry-meals () - cannot make a synonym of viennoiseries for en:viennoiseries
ERROR - en:pastries already is a synonym of pastries (en:pastries) - cannot make a synonym of viennoiseries for en:viennoiseries
```

**Comment by bernardlemoullec** [comment]
> en:viennoiseries



> ```
> Errors in the categories taxonomy definition:
> ERROR - fr:soupe already is a synonym of soupes (en:soups) - cannot make a synonym of soupe for en:soups
> ERROR - fr:quiches already is a synonym of tartes-salees (en:quiches) - cannot make a synonym of quiches for en:quiches
> ERROR - en:puff-pastry already is a synonym of puff-pastry-meals () - cannot make a synonym of viennoiseries for en:viennoiseries
> ERROR - en:pastries already is a synonym of pastries (en:pastries) - cannot make a synonym of viennoiseries for en:viennoiseries
> ```

Merci pour l'info, corrigé sur le prochain commit

**Comment by teolemon** [comment]
Errors in the categories taxonomy definition:
ERROR - fr:cotelette-d-agneau-grillee already is a synonym of cotes-d-agneau-grillees (en:grilled-lamb-cutlet) - cannot make a synonym of cotelette-d-agneau-grillee for en:grilled-lamb-cutlet
ERROR - fr:cotelette-d-agneau-crue already is a synonym of cotes-d-agneau-crues (en:raw-lamb-cutlet) - cannot make a synonym of cotelette-d-agneau-crue for en:raw-lamb-cutlet
ERROR - en:pastries already is a synonym of puff-pastry-meals (en:pastries) - cannot make a synonym of pastries for en:pastries

**Comment by teolemon** [comment]
Errors in the categories taxonomy definition:
ERROR - fr:eau-minerale-gazeuse already is a synonym of eaux-gazeuses () - cannot make a synonym of eaux-minerales-gazeuses for en:carbonated-mineral-waters
ERROR - fr:poudre-cacaotee-sucree-pour-boisson-enrichie-en-vitamines-et-mineraux already is a synonym of poudre-cacaotee-sucree-pour-boisson-enrichie-en-vitamines-et-mineraux (en:cocoa-powder-for-beverages-with-sugar-fortified-with-vitamins-and-chemical-elements) - cannot make a synonym of poudre-maltee-sucree-pour-boisson-enrichie-en-vitamines-et-mineraux for en:malted-cocoa-with-sugar
ERROR - fr:poudre-au-chocolat-sucree-pour-boisson-enrichie-en-vitamines-et-mineraux already is a synonym of poudre-cacaotee-sucree-pour-boisson-enrichie-en-vitamines-et-mineraux () - cannot make a synonym of poudre-maltee-sucree-pour-boisson-enrichie-en-vitamines-et-mineraux for en:malted-cocoa-with-sugar
ERROR - en:dairy-specialty-with-fruits-and-sugar already is a synonym of fermented-milk-yogurt-with-fruits-and-sugar () - cannot make a synonym of yogurt-with-fruits-and-sugar for en:yogurt-with-fruits-and-sugar
ERROR - fr:specialite-laitiere-aromatisee-sucree already is a synonym of yaourts-aromatises () - cannot make a synonym of yaourt-aromatise-sucre for en:yogurt-with-fruits-and-sugar
ERROR - fr:yaourt-nature already is a synonym of yaourts-nature () - cannot make a synonym of yaourts-natures for en:plain-yogurts
ERROR - fr:specialite-laitiere-nature already is a synonym of yaourts-nature () - cannot make a synonym of yaourts-natures for en:plain-yogurts
ERROR - fr:sorbets already is a synonym of glaces-et-sorbets (en:sorbets) - cannot make a synonym of sorbets for en:sorbets
ERROR - fr:glaces-a-l-eau already is a synonym of glaces-et-sorbets (en:ice-pops) - cannot make a synonym of glaces-a-l-eau for en:ice-pops

**Comment by bernardlemoullec** [comment]
Il y a quand même quelques étrangetés dans les granularités CIQUAL...

ciqual_food_code:en:19600
ciqual_food_name:en:Yoghurt, plain (average)
ciqual_food_name:fr:Yaourt ou spécialité laitière nature (aliment moyen)

ciqual_food_code:en:19593
ciqual_food_name:en:Yogurt, fermented milk or dairy specialty, plain
ciqual_food_name:fr:Yaourt, lait fermenté ou spécialité laitière, nature

**Comment by VaiTon** [comment]
@bernardlemoullec can you merge?

**Comment by bernardlemoullec** [comment]
Hello @VaiTon , it seems I cannot merge directly my branch with Master myself, since Github tells me "This branch has conflicts that must be resolved" (the conflicting file being, obviously, taxonomies/categories.txt).
What am I supposed to do?

**Comment by VaiTon** [comment]
> 
> 
> Hello @VaiTon , it seems I cannot merge directly my branch with Master myself, since Github tells me "This branch has conflicts that must be resolved" (the conflicting file being, obviously, taxonomies/categories.txt).
> What am I supposed to do?

I meant merge master so that the conflict goes away. Did you add the lines manually?

**Comment by bernardlemoullec** [comment]
> > Hello @VaiTon , it seems I cannot merge directly my branch with Master myself, since Github tells me "This branch has conflicts that must be resolved" (the conflicting file being, obviously, taxonomies/categories.txt).
> > What am I supposed to do?
> 
> I meant merge master so that the conflict goes away. Did you add the lines manually?

Yes, that is basically the intent behind this pull request (update the taxonomies/categories.txt with CIQUAL values coming from https://docs.google.com/spreadsheets/d/0B2Z-7cB-sMWOY0hybHNoY1pqaXoyMi1nblI1V1g0aTlvVXFF)

**Comment by bernardlemoullec** [comment]
Done (https://github.com/openfoodfacts/openfoodfacts-server/pull/3437/commits/8800c6cb9a36478afb1ee3600d9618879eefc46e)

File: taxonomies/categories.txt [hunk 2]
```diff
@@ -24434,13 +24786,16 @@ ca:Barretes de xocolata
 de:Schokoriegel, Schokoladenriegel
 es:Barritas de chocolate
 fi:suklaapatukat
-fr:Barres chocolatées, barres au chocolat
+fr:Barres chocolatées, barres au chocolat, Chocolat en tablette
```
Comment (teolemon) [inline taxonomies/categories.txt:24789]: @moon-rabbitOFF

**Pull request merged**